gormigrate.go

```diff
@@ -458,8 +458,7 @@ func (g *Gormigrate) unknownMigrationsHaveHappened() (bool, error) {
 }
 
 func (g *Gormigrate) insertMigration(id string) error {
-	record := map[string]interface{}{g.options.IDColumnName: id}
-	return g.tx.Table(g.options.TableName).Create(record).Error
+	return g.tx.Exec("INSERT INTO " + g.options.TableName + " (" + g.options.IDColumnName + ") VALUES ('" + id + "')").Error
```

Comment:
This can lead to SQL Injection and prevent id to have single quote inside. Am I missing something ?

Reply:
It's not much of a PR, more like the draft of a possible fix to get some feedback

Reply:
Oh I see ! No problem, I was just looking as I saw your PR just after I did one ahah.